packages/clients/textLocator/src/services.ts

```diff
@@ -0,0 +1,104 @@
+export const openStreetMap = 'openStreetMap'
+export const openSeaMap = 'openSeaMap'
+export const mdiSeaNames = 'mdiSeaNames'
+export const wmtsTopplusOpenWeb = 'wmtsTopplusOpenWeb'
+export const wmtsTopplusOpenWebGrey = 'wmtsTopplusOpenWebGrey'
+export const wmtsTopplusOpenLight = 'wmtsTopplusOpenLight'
+export const wmtsTopplusOpenLightGrey = 'wmtsTopplusOpenLightGrey'
+export const aerial = 'aerial'
+
+export const idRegister = [
+  openStreetMap,
+  openSeaMap,
+  mdiSeaNames,
+  wmtsTopplusOpenWeb,
+  wmtsTopplusOpenWebGrey,
+  wmtsTopplusOpenLight,
+  wmtsTopplusOpenLightGrey,
+  aerial,
+]
+
+const layerNames = {
+  [wmtsTopplusOpenWeb]: 'web',
+  [wmtsTopplusOpenWebGrey]: 'web_grau',
+  [wmtsTopplusOpenLight]: 'web_light',
+  [wmtsTopplusOpenLightGrey]: 'web_light_grau',
+}
+
+export const services = [
+  ...[
+    wmtsTopplusOpenLight,
+    wmtsTopplusOpenLightGrey,
+    wmtsTopplusOpenWeb,
+    wmtsTopplusOpenWebGrey,
+  ].map((id) => ({
+    id,
+    capabilitiesUrl:
+      'https://sgx.geodatenzentrum.de/wmts_topplus_open/1.0.0/WMTSCapabilities.xml',
+    urls: 'https://sgx.geodatenzentrum.de/wmts_topplus_open',
+    optionsFromCapabilities: true,
+    tileMatrixSet: 'EU_EPSG_25832_TOPPLUS',
+    typ: 'WMTS',
```

Comment:
Sure, but that is not an effort for this pull request. I suggest taking this as a note elsewhere, and further suggest it may be done by writing the documentation right into the masterportalAPI.